packages/reports/addon/templates/components/report-actions/multiple-format-export.hbs

```diff
@@ -9,16 +9,19 @@
       <dd.content>
         <ul class="multiple-format-export__dropdown">
           {{#each this.exportFormats as |exportFormat|}}
-            <li>
-              <NaviIcon @icon={{exportFormat.icon}} />
-              <a
-                download={{dasherize this.filename}}
-                href={{await exportFormat.href}}
-                {{on "click" (queue (fn this.notify exportFormat.type) (fn this.close dd))}}
-              >
-                {{exportFormat.type}}
-              </a>
-            </li>
+            {{! @report.isNew is falsey with dashboards so this works as expected in dashboards}}
+            {{#if (not (and @report.isNew exportFormat.requiresSaved))}}
```

Comment:
It might be a better experience to show all the formats and disable the invalid ones with a tooltip that says the report needs to be saved first

Reply:
Yeah thought about this after I opened up PR so on same page here. I just pushed an update.